TurkishFilter/sections/specific.txt

```diff
@@ -28,6 +28,9 @@
 ! https://github.com/AdguardTeam/AdguardFilters/issues/37555
 !+ PLATFORM(ext_ff, ext_opera, ios, ext_android_cb, ext_safari, ext_ublock)
 merlininkazani.com##div[id^="div-gpt-ad"]
+! https://github.com/AdguardTeam/AdguardFilters/issues/39334
+!+ NOT_PLATFORM(ext_android_cb, ios, ext_safari)
+dizilara.org##.invert
```

Comment:
does not it prevent to open video before end of ad?

Reply:
Only on platforms without js-support. I exchanged rules to css variant.